mmv1/third_party/terraform/services/logging/resource_logging_project_sink.go

```diff
@@ -57,7 +62,18 @@ func resourceLoggingProjectSinkCreate(d *schema.ResourceData, meta interface{})
 	id, sink := expandResourceLoggingSink(d, "projects", project)
 	uniqueWriterIdentity := d.Get("unique_writer_identity").(bool)
 
-	_, err = config.NewLoggingClient(userAgent).Projects.Sinks.Create(id.parent(), sink).UniqueWriterIdentity(uniqueWriterIdentity).Do()
+	customWriterIdentity := d.Get("custom_writer_identity").(string)
+
+	client := config.NewLoggingClient(userAgent).Projects.Sinks.Create(id.parent(), sink)
```

Comment:
```suggestion
	projectSinkCreateRequest := config.NewLoggingClient(userAgent).Projects.Sinks.Create(id.parent(), sink)
```

nit: This is a request object rather than a client

Reply:
Ah- you'll need to refactor the references as well!

Reply:
yes, working on it.